xml/compute.xml

```diff
@@ -759,12 +759,12 @@
    <title>memkind &mdash; heap manager for heterogeneous memory platforms and mixed memory policies</title>
    <para>
     The <emphasis>memkind</emphasis> library is a user-extensible heap manager
-    built on top of <literal>jemalloc</literal> which enables control of memory
-    characteristics and a partitioning of the heap between kinds of memory. The
-    kinds of memory are defined by operating system memory policies that have
-    been applied to virtual address ranges. Memory characteristics supported by
-    <literal>memkind</literal> without user extension include control of NUMA
-    and page size features.
+    built on top of <emphasis>jemalloc</emphasis> which enables control of
+    memory characteristics and a partitioning of the heap between kinds of
+    memory. The kinds of memory are defined by operating system memory policies
+    that have been applied to virtual address ranges. Memory characteristics
+    supported by <package>memkind</package> without user extension include
+    control of NUMA and page size features.
```

Comment:
I can't quite put my finger at it, but I think this may need a bit more clarifications. E.g., how are those "kinds" of memory different etc. -- I realize that the final sentence may answer this, but I am not clever enough to understand it.

Reply:
I think if we do not fully understand it, we should leave well alone.

Reply:
I forgot to tag @e4t here. :)